README.md

```diff
@@ -41,6 +41,20 @@ The default font family, color and line/character spacing will look like this:
 
 ![screenshot of paragraph text](/screenshots/paragraph.png?raw=true)
 
+### Compact Body
+
+The compact body class is a smaller version of the default body class, for use in areas that prefer to be conservative with the amount of real estate used by text.
+
+The compact style is not recommended for blocks of readable text, particularly in paragraph form. Rather, it is best employed for brief informative text or calls to action.
```

Comment:
I tend to hesitate putting design practices into these readme docs.  It's subject to change and it's one more thing to fall out of date or become inconsistent (multiple sources of truth).  I usually stick to "how do I technically use this thing".